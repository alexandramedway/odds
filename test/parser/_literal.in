do 42
do 42.1
do "forty-two"
do true
do false
do void
