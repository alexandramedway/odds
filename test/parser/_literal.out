[
    Expr(Int_lit(42)) ;
    Expr(Float_lit(42.1)) ;
    Expr(String_lit(forty-two)) ;
    Expr(Bool_lit(true)) ;
    Expr(Bool_lit(false))
]
